src/env.ts

```diff
@@ -32,7 +33,7 @@ export function defineEnv(opts: CreateEnvOptions = {}): {
 
   const resolvedEnv = env(...presets);
 
-  return { env: resolvedEnv };
+  return { env: resolvedEnv, presets };
```

Comment:
What would the usage look like?

If we usually need both env and preset maybe:
`const presets = definePreset(opts: CreateEnvOptions = {})` and `env(...presets)` would work too.

_Mostly thinking out loud here_

Reply:
`const { env } = defineEnv(opts)`. `env` is current "builder config" but we can add more utils, meta, etc top level without breaking this. having all on same level might be tricky to maintain long term.